src/libraries/System.Configuration.ConfigurationManager/src/System.Configuration.ConfigurationManager.csproj

```diff
@@ -13,8 +13,14 @@
   <!-- DesignTimeBuild requires all the TargetFramework Derived Properties to not be present in the first property group. -->
   <PropertyGroup>
     <IsPartialFacadeAssembly Condition="$([MSBuild]::GetTargetFrameworkIdentifier('$(TargetFramework)')) == '.NETFramework'">true</IsPartialFacadeAssembly>
+    <!-- ILLinker settings -->
+    <ILLinkDirectory>$(MSBuildThisFileDirectory)ILLink\</ILLinkDirectory>
```

Comment:
You shouldn't need this. It is already set in 

https://github.com/dotnet/runtime/blob/340d1f22ef3b29b8c6b5369b68e73e5f7afea79c/eng/illink.targets#L10